hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/hive/HoodieCombineHiveInputFormat.java

```diff
@@ -872,7 +873,7 @@ public void createPool(JobConf conf, PathFilter... filters) {
         job.set("hudi.hive.realtime", "true");
         InputSplit[] splits;
         if (hoodieFilter) {
-          HoodieParquetInputFormat input = new HoodieParquetRealtimeInputFormat();
+          HoodieParquetRealtimeInputFormat input = new HoodieParquetRealtimeInputFormat();
```

Comment:
Is this change strictly necessary for this pr

Reply:
This is a behavior change.. ccc @n3nash  to confirm if this is ok

Reply:
Reverted.